### PR TITLE
Add and use our own CVE API

### PR DESCRIPTION
**Pull request by dekkers**
Add cveapi script that downloads CVE details from the NVD API and stores it in a directory to be made available by web server such as Nginx.

Change kat_cve_finding_types boefje to support fetching this data from https://cve.openkat.dev and make the URL configurable using a setting.

Update normalizer to the new CVE API format instead of the old deprecated one.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: boefjes/boefjes/plugins/kat_cve_finding_types/schema.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "title": "Arguments",
+    "type": "object",
+    "properties": {
+        "PORTS": {
+            "title": "CVEAPI_URL",
+            "maxLength": 2048,
+            "type": "string",
+            "description": "URL of the CVE API, defaults to https://cve.openkat.dev/v1"
```
Comment (underdarknl) [inline boefjes/boefjes/plugins/kat_cve_finding_types/schema.json:9]: You could add the "default" label to show the user which url will be the default. We might not render it yet, but we will.

**Review by praseodym — APPROVED** [review]

File: boefjes/boefjes/plugins/kat_cve_finding_types/schema.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "title": "Arguments",
+    "type": "object",
+    "properties": {
+        "PORTS": {
```
Comment (Darwinkel) [inline boefjes/boefjes/plugins/kat_cve_finding_types/schema.json:5]: PORTS property?

File: cveapi/pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+[tool.black]
+target-version = ["py311"]
```
Comment (Darwinkel) [inline cveapi/pyproject.toml:2]: `target-version = ["py38", "py39", "py310", "py311"]` per https://github.com/psf/black/issues/751

**Review by ammar92 — DISMISSED** [review]

**Comment by Darwinkel** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [ ] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* CVE findings are run and created successfully
image


### What doesn't work:
* PORTS argument should be named differently
image
* Attempting to change the settings of the CVE boefje results in the below error:
image

File: cveapi/debian/copyright [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: kat-cveapi
+Upstream-Contact: info@openkat.nl
+Source: https://github.com/minvws/nl-kat-coordination
+
+Files: *
+Copyright: Ministry of Public Health, Wellbeing and Sports
```
[the remark this answers is not in the document]
Comment (dekkers) [reply cveapi/debian/copyright]: Fixed it also in the place I copied it from.

File: cveapi/pyproject.toml (hunk 3 above)
Comment (dekkers) [reply cveapi/pyproject.toml]: It also doesn't hurt to have this consistent with the rest, so I changed it.

File: cveapi/debian/copyright (hunk 4 above)
[the remark this answers is not in the document]
Comment (praseodym) [reply cveapi/debian/copyright]: This string is in a few more places:
```
> rg -C1 "Public Health"
rocky/onboarding/templates/step_1_introduction.html
15-            OpenKAT is the "Kwetsbaarheden Analyse Tool" (Vulnerabilities Analysis Tool). An Open-Source-project developed by the
16:            Ministry of Public Health, Wellbeing and Sports to make your and our world a safer place.
17-         {% endblocktranslate %}

rocky/rocky/locale/pap/LC_MESSAGES/django.po
2051-"OpenKAT is the \"Kwetsbaarheden Analyse Tool\" (Vulnerabilities Analysis "
2052:"Tool). An Open-Source-project developed by the Ministery of Public Health, "
2053-"Wellbeing and Sports to make your and our world a safer place."

rocky/rocky/locale/nl/LC_MESSAGES/django.po
2054-"OpenKAT is the \"Kwetsbaarheden Analyse Tool\" (Vulnerabilities Analysis "
2055:"Tool). An Open-Source-project developed by the Ministery of Public Health, "
2056-"Wellbeing and Sports to make your and our world a safer place."
```

**Review by underdarknl — APPROVED** [review]

**Pull request merged**